spec.md

```diff
@@ -164,49 +205,105 @@ echo "SPDX-License-Identifier: MIT" > file.txt
+  both contain Information for the same Covered File. Available values are:
+
+  - `closest`, the default value when `precedence` is not defined. This is an
+    instruction to associate the Copyright and Licensing Information inside of
+    the Covered Files, if available. If no such Information is found, then the
+    Information inside the table of the closest `REUSE.toml` that covers the
+    File is associated. This algorithm is applied separately for copyright and
+    for licensing. If a table for the same File in a closer `REUSE.toml` file
+    has the `override` precedence, then that precedence is applied, and
+    `closest` is ignored. This is effectively a fallback.
+
+  - `aggregate`. This is an instruction to always associate the Copyright and
+    Licensing Information defined in the table with the table's Covered Files.
+    Subsequently, the `closest` logic is also applied.
 
-Snippets may nest, and this is denoted by having
-`SPDX-SnippetBegin`/`SPDX-SnippetEnd` pairs within other pairs, in the same way
-that parentheses nest in mathematical expressions. In the case of nested
-snippets, the SPDX file tags are considered to apply to the inner-most snippet.
+  - `override`. This is an instruction to associate the Copyright and Licensing
+    Information defined in the table with the table's Covered Files, and to
+    ignore any other Information that is closer to the Files. The table in the
+    `REUSE.toml` that is the closest to the root of the Project is
+    authoritative.
 
-### DEP5
+- `SPDX-FileCopyrightText` (OPTIONAL), a string or list of strings. Each string
+  MUST be a Copyright Notice to be associated with the table's Covered Files.
+  The prefix of the Copyright Notice MAY be omitted.
 
-Alternatively, Copyright and Licensing Information MAY be associated with a
-file through a DEP5 file. The intended use case of this method is large
-directories where including a comment header in each file (or in `.license`
-companion files) is impossible or undesirable.
+- `SPDX-License-Identifier` (OPTIONAL), a string or list of strings. Each string
+  MUST be a valid SPDX License Expression describing the licensing of the
+  table's Covered Files.
+
+Although the keys to associate Copyright and Licensing Information with the
+Covered File are OPTIONAL, the complete Information MUST still be associated
+with the File in some fashion.
+
+If a Covered File is covered by multiple `[[annotations]]` tables in the same
+`REUSE.toml` file, then exclusively the last matching table in the file is used
+for that Covered File.
+
+An example of a `REUSE.toml` file:
+
+```toml
+version = 1
+
+[[annotations]]
+path = ["po/*.po", "po/*.pot"]
+precedence = "aggregate"
+SPDX-FileCopyrightText = "2019 Translation Company"
+SPDX-License-Identifier = "GPL-3.0-or-later"
+```
+
+### DEP5 (Deprecated)
+
+Copyright and Licensing Information MAY be associated with a file through a DEP5
+file, but you SHOULD create a `REUSE.toml` file instead.
```

Comment:
Outside of the title, it should be good to include in the text that this functionality is deprecated.

```suggestion
file, but you SHOULD create a `REUSE.toml` file instead as the DEP5 file is
considered deprecated.
```

Reply:
Ack, added a comment.